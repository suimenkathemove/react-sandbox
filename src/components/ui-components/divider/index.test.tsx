import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import 'jest-styled-components';
import { Divider } from './';

const setup = () => render(<Divider {...defaultProps} />);

describe('Divider', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
