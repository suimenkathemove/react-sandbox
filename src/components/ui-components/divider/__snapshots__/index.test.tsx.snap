// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider render 1`] = `
<DocumentFragment>
  .c0 {
  border-color: gray;
}

<hr
    class="c0"
  />
</DocumentFragment>
`;
